src/libraries/Microsoft.PowerFx.Core/Public/IntellisenseOperations.cs

```diff
@@ -74,14 +74,20 @@ public bool ValidateInvocation(string functionName, IReadOnlyList<TexlNode> args
             foreach (var fnc in fncs)
             {
                 var result =
-                    fnc.CheckInvocation(
-                        _checkResult._binding,
+                    fnc.CheckTypes(
+                        _checkResult._binding.CheckTypesContext,
                         args.ToArray(),
                         types,
                         _checkResult._binding.ErrorContainer,
                         out var retDType,
                         out _);
 
+                fnc.CheckSemantics(
+                    _checkResult._binding,

```

Comment:
This code appears to only care about the return type, and since it's intellisense, the errors don't matter. We can probably safely skip CheckSemantics?

Reply:
Fixed